From 21dbd1c42b290dec2ce61507b2b6d57cca4e8766 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Tue, 20 Dec 2011 22:19:45 +0000
Subject: cpan/CPAN: add NAME headings in modules with POD

This fixes the Debian Lintian warning about missing NAME sections in
manpages.

Bug: https://rt.cpan.org/Ticket/Display.html?id=73396
Bug-Debian: http://bugs.debian.org/650448

Patch-Name: fixes/manpage_name_CPAN.diff
---
 cpan/CPAN/lib/CPAN/Debug.pm        |    4 ++++
 cpan/CPAN/lib/CPAN/HandleConfig.pm |    6 ++++++
 cpan/CPAN/lib/CPAN/Queue.pm        |    4 ++++
 cpan/CPAN/lib/CPAN/Tarzip.pm       |    4 ++++
 4 files changed, 18 insertions(+)

diff --git a/cpan/CPAN/lib/CPAN/Debug.pm b/cpan/CPAN/lib/CPAN/Debug.pm
index 23c4a36..48e394b 100644
--- a/cpan/CPAN/lib/CPAN/Debug.pm
+++ b/cpan/CPAN/lib/CPAN/Debug.pm
@@ -71,6 +71,10 @@ sub debug {
 
 __END__
 
+=head1 NAME
+
+CPAN::Debug - internal debugging for CPAN.pm
+
 =head1 LICENSE
 
 This program is free software; you can redistribute it and/or
diff --git a/cpan/CPAN/lib/CPAN/HandleConfig.pm b/cpan/CPAN/lib/CPAN/HandleConfig.pm
index 09c42ef..a03e165 100644
--- a/cpan/CPAN/lib/CPAN/HandleConfig.pm
+++ b/cpan/CPAN/lib/CPAN/HandleConfig.pm
@@ -6,6 +6,12 @@ use File::Spec ();
 use File::Basename ();
 use Carp ();
 
+=head1 NAME
+
+CPAN::HandleConfig - internal configuration handling for CPAN.pm
+
+=cut 
+
 $VERSION = "5.5003"; # see also CPAN::Config::VERSION at end of file
 
 %can = (
diff --git a/cpan/CPAN/lib/CPAN/Queue.pm b/cpan/CPAN/lib/CPAN/Queue.pm
index e5d88ce..1222b37 100644
--- a/cpan/CPAN/lib/CPAN/Queue.pm
+++ b/cpan/CPAN/lib/CPAN/Queue.pm
@@ -201,6 +201,10 @@ sub reqtype_of {
 
 __END__
 
+=head1 NAME
+
+CPAN::Queue - internal queue support for CPAN.pm
+
 =head1 LICENSE
 
 This program is free software; you can redistribute it and/or
diff --git a/cpan/CPAN/lib/CPAN/Tarzip.pm b/cpan/CPAN/lib/CPAN/Tarzip.pm
index 972df6c..cf8aad4 100644
--- a/cpan/CPAN/lib/CPAN/Tarzip.pm
+++ b/cpan/CPAN/lib/CPAN/Tarzip.pm
@@ -450,6 +450,10 @@ END
 
 __END__
 
+=head1 NAME
+
+CPAN::Tarzip - internal handling of tar archives for CPAN.pm
+
 =head1 LICENSE
 
 This program is free software; you can redistribute it and/or
